types/hyper-function-component/index.d.ts

```diff
@@ -1,13 +1,14 @@
-// Type definitions for hyper-function-component 1.1
+// Type definitions for hyper-function-component 1.2
 // Project: https://hyper-function.com/ponent
```

Comment:
btw, this results in 404:
https://hyper-function.com/ponent
![image](https://user-images.githubusercontent.com/14539/177209437-029eabc0-55d3-4233-9283-932160a04467.png)